### PR TITLE
fix arabica network select

### PR DESCRIPTION
**Pull request by jcstein**
## Overview

This PR:
- [x] changes network selection flags from `--node.network` to `--p2p.network`
- [x] updates `arabica-1` to `arabica-2` in Init responses
- [x] changes network selection args from `arabica-1` back to `arabica`

## Checklist

- [x] New and updated code has appropriate documentation
- [x] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [x] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

### CODE REVIEW

**Comment by jcstein** [comment]
@renaynay mentioned these fixes in a comment, but I cannot seem to find it 😅

File: docs/developers/celestia-node.mdx [hunk 1]
```diff
@@ -289,22 +289,22 @@ Error: nodebuilder/share: interval must be positive
...
+rm -rf ~./celestia-light-arabica-2 && celestia light init --p2p.network arabica
 ```
 
 Your output will look similar to below:
 
 ```sh
-@ ~ % rm -rf ./celestia-light-arabica-1 && celestia light init
-2022-11-01T22:23:19.581+0100	INFO	node	nodebuilder/init.go:20	Initializing Light Node Store over '/Users/joshstein/.celestia-light-arabica-1'
-2022-11-01T22:23:19.582+0100	INFO	node	nodebuilder/init.go:51	Saving config	{"path": "/Users/joshstein/.celestia-light-arabica-1/config.toml"}
+@ ~ % rm -rf ./celestia-light-arabica-2 && celestia light init
```
Comment (YazzyYaz) [inline docs/developers/celestia-node.mdx:298]: delete this line as you're saying "your output" so you're just showing the logs after running this command and you already ran it on line 292

File: docs/developers/celestia-node.mdx [hunk 2]
```diff
@@ -333,7 +333,7 @@ rm -rf ~./celestia-light && celestia light init
 Your output will look similar to below:
 
 ```sh
-@ ~ % rm -rf ./celestia-light-arabica-1 && celestia light init
+@ ~ % rm -rf ./celestia-light && celestia light init
```
Comment (YazzyYaz) [inline docs/developers/celestia-node.mdx:336]: delete this line too, same point as the above.

**Review by YazzyYaz — CHANGES_REQUESTED** [review]

File: docs/developers/celestia-node.mdx (hunk 1 above)
Comment (jcstein) [reply docs/developers/celestia-node.mdx:298]: ✅ done - thank you

File: docs/developers/celestia-node.mdx (hunk 2 above)
Comment (jcstein) [reply docs/developers/celestia-node.mdx:336]: ✅

**Review by renaynay — APPROVED** [review]

**Review by YazzyYaz — APPROVED** [review]

**Comment by YazzyYaz** [comment]
LGTM

**Pull request merged**